maintainers/maintainer-list.nix

```
@@ -2361,6 +2361,12 @@
     github = "CaptainJawZ";
     githubId = 43111068;
   };
+  CardboardTurkey = {
+    name = "Kiran";
```

```suggestion
    name = "Kiran Ostrolenk";
```
I suppose (unless that's intentional to be anonymous, but the last name is in the email anyway).